environments/__prod_envs/vars/versions.yml

```diff
@@ -1,2 +1,3 @@
 webview_version: v0.54.0
 press_version: v8.1.7
+archive_version: 4.15.8
```

Comment:
Should this reflect the latest version update to archive? (It's currently at `4.15.9`)

Reply:
Yes, we should update to the latest version